manifests/m/Microsoft/PIX/21.8.18.1/Microsoft.PIX.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.PIX
+PackageVersion: 21.8.18.1
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+Author: Microsoft
+PackageName: PIX on Windows

```

This needs to match `A`dd or `R`emove `P`rograms
```suggestion
PackageName: Microsoft PIX
```